### ISSUE TITLE
Expose client reports

### ISSUE THREAD

**Issue by rbisol** [getsentry/sentry#42940]
### Problem Statement

Some SDKs submit Client reports with how many events got dropped at the SDK level (beforeSend, network errors, rate-limits...). However, it is not exposed to Sentry users in any form.

Dev docs:
https://develop.sentry.dev/sdk/client-reports/

Some of the SDKs that currently support it: Java, JS, Python, .NET

### Solution Brainstorm

The inbound filters page has a breakdown of how many events were dropped by each filter, the client reports could be displayed the same way. 

A bar graph with how many events were dropped per day, and the bar being divided by reason is a clear way to present the report.

**Comment by brentc** [comment]
Seems like this would belong on the stats page, if anywhere. Re-routing to Enterprise to consider for inclusion.

**Comment by leedongwei** [comment]
Yep, agreed that we should surface this on the stats page. Would love to see this in a sankey diagram.
